packages/website/src/styles/main.scss

```diff
@@ -1,6 +1,6 @@
-@import '~@coveord/plasma-style/dist/css/CoveoStyleGuide.css';
```

Comment:
This path will need to change [here](https://github.com/coveo/plasma/blob/master/packages/website/src/building-blocs/getCodeSandboxLink.tsx#L77) as well